.env

```diff
@@ -1,3 +1,3 @@
 ATS_VERSION=9.2.4
-GO_VERSION=1.23.1
+GO_VERSION=1.22.5
```

Comment:
Looks like you downgraded the Go version? 0b21b43d3483 is unnecessary, you only need c9ab679955a4, which does not affect the Go version at all.